<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(63, 162, 243, 155), stop:1 rgba(255, 255, 255, 255))}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="title">
             <string>Basic Information</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>First Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Midddle Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_23">
                 <property name="text">
                  <string>Last Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Number of Family member</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Blood Group</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Addher Number</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_24">
                 <property name="text">
                  <string>Data of Brith</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_25">
                 <property name="text">
                  <string>Data Of Joing</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_26">
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>Nomine</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="leftMargin">
                <number>20</number>
               </property>
               <property name="rightMargin">
                <number>20</number>
               </property>
               <item>
                <widget class="QLineEdit" name="fname">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>MS Shell Dlg 2</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="placeholderText">
                  <string>First Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="mname">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>MS Shell Dlg 2</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="placeholderText">
                  <string>Middle Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lname">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>MS Shell Dlg 2</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
                 </property>
                 <property name="placeholderText">
                  <string>Last Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="nofc">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>MS Shell Dlg 2</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
                 </property>
                 <property name="placeholderText">
                  <string>Family Member</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="bloodGroup">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>MS Shell Dlg 2</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="placeholderText">
                  <string>Blood Group</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="addherNo">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>MS Shell Dlg 2</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="placeholderText">
                  <string>Addher Number</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="DOB">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>MS Shell Dlg 2</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="placeholderText">
                  <string>Data of Birth</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="DOJ">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>MS Shell Dlg 2</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="placeholderText">
                  <string>Data of Joing</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Nomine">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>MS Shell Dlg 2</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="placeholderText">
                  <string>Nomine</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="title">
             <string>Contact Information</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="label_22">
                 <property name="text">
                  <string>Mobile No</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Mobile No</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_21">
                 <property name="text">
                  <string>Email</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="leftMargin">
                <number>20</number>
               </property>
               <property name="rightMargin">
                <number>20</number>
               </property>
               <item>
                <widget class="QLineEdit" name="Mobile1">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>MS Shell Dlg 2</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="placeholderText">
                  <string>Mobile Number</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Mobile2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>MS Shell Dlg 2</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="placeholderText">
                  <string>Mobile Number</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="email">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>MS Shell Dlg 2</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="placeholderText">
                  <string>Email</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,4,4,0">
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Dashboard</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Manage</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>View</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Exit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="3">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>Send Mail</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="add_info">
         <property name="text">
          <string>Add Information</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLabel" name="error">
         <property name="styleSheet">
          <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255, 0, 0)</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>fname</tabstop>
  <tabstop>mname</tabstop>
  <tabstop>lname</tabstop>
  <tabstop>nofc</tabstop>
  <tabstop>bloodGroup</tabstop>
  <tabstop>addherNo</tabstop>
  <tabstop>DOB</tabstop>
  <tabstop>DOJ</tabstop>
  <tabstop>Nomine</tabstop>
  <tabstop>Mobile1</tabstop>
  <tabstop>Mobile2</tabstop>
  <tabstop>email</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
